hipo_drf_exceptions/handlers.py

```diff
@@ -17,8 +17,22 @@ def get_fallback_message(exception):
                 # Return first non-empty value in the list. https://github.com/Hipo/hipo-drf-exceptions/issues/8
                 return get_fallback_message(item)
     elif isinstance(exception, dict):
+        # Get message from the first key.
         first_key = next(iter(exception))
-        return get_fallback_message(exception[first_key])
+        message = exception[first_key]
+
+        # Check if the message contains the field name.
```

Comment:
IMO, This is not the responsibility of the package. `first_name: This field is required` would be enough message for fallback message.

Reply:
I think so too. So, should `f"{human_readable_key}: {message}"` be returned in every case?

Reply:
I think so too. So, should f"{human_readable_key}: {message}" be returned in every case?

Reply:
So, Why we check message contains field name?